pkg/kubelet/dockertools/docker.go

```diff
@@ -63,9 +64,9 @@ type DockerInterface interface {
 	StartContainer(id string, hostConfig *docker.HostConfig) error
 	StopContainer(id string, timeout uint) error
 	RemoveContainer(opts docker.RemoveContainerOptions) error
-	InspectImage(image string) (*docker.Image, error)
-	ListImages(opts docker.ListImagesOptions) ([]docker.APIImages, error)
-	PullImage(opts docker.PullImageOptions, auth docker.AuthConfiguration) error
+	InspectImage(image string) (*dockertypes.ImageInspect, error)
+	ListImages(opts dockertypes.ImageListOptions) ([]dockertypes.Image, error)
+	PullImage(imageID string, tag string, auth dockertypes.AuthConfig) error
```

Comment:
We may need something like this:
https://github.com/docker/engine-api/pull/162/files#diff-dbeb877b7334f75bc4c9063000523d71R22
So that in the future if the `dockertypes.ImagePullOptions` changed, we don't need to touch too much code.